pattern/src/simple.rs

```diff
@@ -34,6 +39,7 @@ impl<T: Clone, P: Pattern<T>> Pattern<T> for PBind<P> {
     }
 }
 
+/// Matches `Net`s and `Value`s that are constants.
```

Comment:
This is a bit ambiguous, I'd go for something like this that makes it clear how undefs are treated:

```suggestion
/// Matches `Net`s and `Value`s that do not refer to cells (consist entirely of zero, one, and undef nets).
```